documentation/openvino.md

```diff
@@ -0,0 +1,95 @@
+Build a Yocto Image with OpenVINO™ toolkit
+==========================================
+
+Follow the [Yocto Project official documentation](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html#compatible-linux-distribution) to set up and configure your host machine to be compatible with BitBake.
+
+## Step 1: Set Up Environment
+
+1. Clone the repositories.
+
+```
+      git clone https://git.yoctoproject.org/git/poky
+      git clone https://github.com/openembedded/meta-openembedded
+      git clone https://git.yoctoproject.org/git/meta-intel
```

Comment:
this documentation is specific to meta-intel and doesn't need to be brought in.

Reply:
removed and pushed the changes